user-allocations/.gitignore

```diff
@@ -0,0 +1 @@
+csv
```

Comment:
Is this supposed to ignore a folder called `csv` or what is the purpose of this one?

Reply:
that's a forgotten leftover from a previous version. removed.